examples/stable_diffusion_v2/README.md

```diff
@@ -165,8 +167,46 @@ By setting empty prompt (`--prompt=""`), the masked part will be auto-filled to
 
 ### Text-guided Image-to-Image
 
-Coming soon
+This pipeline uses a finetuned version of Stable Diffusion 2.1, which can be used to create image variations (image-to-image). Here we provide two models `2.1-unclip-l` and `2.1-unclip-h`, which uses the pretrained Clip Image embedder and Open Clip Image embedder separately. User can use the `-v` argument to decide which model to use. The amount of image variation can be controled by the noise injected to the image embedding, which can be input by the `--noise_level` argument. (0 means no noise, 1000 full noise).
```

Comment:
CLIP, Users, controlled, ... please use grammarly / GPT to correct solecism.

Reply:
Updated, thx